python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/application_traffic_recognition.schema.yml

```diff
@@ -124,6 +124,7 @@ keys:
                   description: Application name.
       application_profiles:
         type: list
+        primary_key: name  # TODO: Verify with gmuloc if this can be added safely.
```

Comment:
@gmuloc can you help me decide here? I believe the name key should be required and unique.

Reply:
yes it can be added safely